src/policy.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2014 The Bitcoin developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_POLICY_H
+#define BITCOIN_POLICY_H
+
+#include "consensus/consensus.h"
+#include "script/standard.h"
+
+#include <map>
+#include <string>
+
+class CFeeRate;
+
+/** Default for -blockmaxsize and -blockminsize, which control the range of sizes the mining code will create **/
+static const unsigned int DEFAULT_BLOCK_MAX_SIZE = 750000;
+static const unsigned int DEFAULT_BLOCK_MIN_SIZE = 0;
+/** Default for -blockprioritysize, maximum space for zero/low-fee transactions **/
+static const unsigned int DEFAULT_BLOCK_PRIORITY_SIZE = 50000;
+/** The maximum size for transactions we're willing to relay/mine */
+static const unsigned int MAX_STANDARD_TX_SIZE = 100000;
+/** Maximum number of signature check operations in an IsStandard() P2SH script */
+static const unsigned int MAX_P2SH_SIGOPS = 15;
+/** The maximum number of sigops we're willing to relay/mine in a single tx */
+static const unsigned int MAX_STANDARD_TX_SIGOPS = MAX_BLOCK_SIGOPS/5;
+/**
+ * Standard script verification flags that standard transactions will comply
+ * with. However scripts violating these flags may still be present in valid
+ * blocks and we must accept those blocks.
+ */
+static const unsigned int STANDARD_SCRIPT_VERIFY_FLAGS = MANDATORY_SCRIPT_VERIFY_FLAGS |
+                                                         SCRIPT_VERIFY_STRICTENC |
+                                                         SCRIPT_VERIFY_MINIMALDATA |
+                                                         SCRIPT_VERIFY_NULLDUMMY |
+                                                         SCRIPT_VERIFY_DISCOURAGE_UPGRADABLE_NOPS |
+                                                         SCRIPT_VERIFY_CLEANSTACK;
+/** For convenience, standard but not mandatory verify flags. */
+static const unsigned int STANDARD_NOT_MANDATORY_VERIFY_FLAGS = STANDARD_SCRIPT_VERIFY_FLAGS & ~MANDATORY_SCRIPT_VERIFY_FLAGS;
+
+/** GLOBALS: These variables are supposed to become CStandardPolicy attributes */
+
+extern bool fIsBareMultisigStd;
+extern CFeeRate minRelayTxFee;
+
+/** Abstract interface for Policy */
+class CPolicy
+{
+public:
+    virtual void InitFromArgs(const std::map<std::string, std::string>&) = 0;
+    virtual bool ValidateScript(const CScript&, txnouttype&) const = 0;
```

Comment:
I know, that can be done later as shown in https://github.com/jtimon/bitcoin/compare/luke_policy2 (a rebased version of your stuff).
That's not the only possible interface, it may not be the best (specially since we lack a criterion of quality for policy interfaces) and, again, the interface doesn't need to be perfect from the beginning. 
The priority is encapsulating policy-specific code. The fastest way to do it is by doing the most obvious code movements first. 
Let's do the uncontroversial code movements first, trying to get it perfect from the beginning may result in having this delayed indefinitely.